.github/workflows/cypress-Run-all-tests-no-cache.yml

```diff
@@ -112,4 +144,4 @@ jobs:
         uses: actions/upload-artifact@master
         with:
           name: videos-${{ matrix.type }}-${{ matrix.browser }}
-          path: cypress/videos
+          path: cypress/videos
```

Comment:
The file is missing an empty last trail.

Reply:
Right, will solve it